README.md

```diff
@@ -1,2 +1,26 @@
 # hw01
 Repository for homework 01
+
+# Deblina Mukherjee 
+
+I'm a second year in the College at UChicago majoring in **sociology** (with a specialization in *computational social science*) and **statistics**. 
+
+
+![](https://raw.githubusercontent.com/deblnia/deblnia.github/master/pic3.jpg)
+
+## Research 
+
+I spent the early part of the summer working with Karthick Ramakrishnan at [AAPI Data](aapidata.com), and I did research with [Professor Robert Gulotty](http://home.uchicago.edu/~gulotty/index.html) focusing on political economy in American Rust Belts for the rest of the summer. I've been involved with the [Knowledge Lab](https://www.knowledgelab.org/) since my first year, and currently have a paper under review at [Brown's JPPE](https://www.brownjppe.com/).  
+
+## Comedy 
+
+I do two comedy activities on-campus: 
+
+***1. Occam's Razor Comedy Improv*** 
```

Comment:
Another way to format this list:

```
1. Occam's Razor Comedy Improv
    I practice and perform improvisational comedy.
1. The Chicago Shady Dealer
    I both write and compile layout for this publication. 
```

1. Occam's Razor Comedy Improv
    I practice and perform improvisational comedy.
1. The Chicago Shady Dealer
    I both write and compile layout for this publication.